### ISSUE TITLE
Image Accept: headers in the spec prevent conneg

### ISSUE THREAD

**Issue by wolfbeast** [whatwg/fetch#877]
The fetch spec pretty much mandates that everyone uses:
`image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5` for image requests.
This should be adapted to give user agents the possibility to include image formats that might not be universally accepted but preferred, e.g. `image/webp` or `image/apng` or `image/jxr`, and to exclude formats that might not be supported or preferred (e.g. excluding svg support for agents on systems where vector graphics are disproportionately expensive to process), to allow server-driven [conneg](https://en.wikipedia.org/wiki/Content_negotiation) to happen with servers that support multiple other formats.

As it stands, the spec doesn't allow for this kind of required flexibility, and users agents would not be spec compliant if they indicate their capabilities to servers.

**Comment by annevk** [comment]
This is #274, no?

**Comment by wolfbeast** [comment]
Hmm I didn't think that issue describes the exact problem, but to be fair I didn't read every single comment. There is no confusion, after all, but the enforced headers don't provide needed flexibility.

**Comment by mattatobin** [comment]
And this was closed without comment, why exactly?

**Comment by annevk** [comment]
It's a duplicate of the aforementioned issue.

**Comment by bathos** [comment]
_Just a note for others who may land here in the future:_

If you’re here due to being puzzled by the seemingly “hard-coded” per-destination `accept` header values, and perhaps have remained puzzled even after following the link to the duplicate (which acknowledges the confusion, but may not immediately dispel it): the key “ohhhh” thing you (and me until, a moment ago) are likely missing is one thread link deeper, [here](https://github.com/whatwg/fetch/issues/43#issuecomment-98959907):

> Well, it already says "should" so user agents have some leeway [...]

In other words, because the verb is “should,” Fetch isn’t really preventing agents from setting other values (or none) and agents that currently send other values (which I think is all of em, in practice?) are compliant.

**Comment by mattatobin** [comment]
As a reminder to WhatWG Contributors: I am no longer a member of or associated with Moonchild Productions (Pale Moon) and do not necessarily hold the same views today that I did when previously commenting on WhatWG venues.
